src/main/java/seedu/address/logic/commands/UnassignCommand.java

```diff
@@ -14,6 +14,7 @@
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.assignment.Assignment;
+//import seedu.address.model.assignment.exceptions.AssignmentNotFoundException;
```

Comment:
Maybe this can be removed if it's not needed.

Reply:
Good catch, will delete!